testData/preferences/snapset_1a.json

```diff
@@ -4,9 +4,7 @@
             "gpii-default": {
                 "name": "Default preferences",
                 "preferences": {
-                    "http://registry.gpii.net/applications/com.microsoft.windows.screenDPI": {
-                        "screen-dpi": 1.25
-                    },
+                    "http://registry.gpii.net/common/screenDPI": 1.25,
```

Comment:
I think the units of the common term should be sane ones, that is the actual DPI value, in this case 125 - we just need to add a linearScale transform in the solutions registry entry for the windows solution in that case